movies-manager/.gitignore

```diff
@@ -0,0 +1,5 @@
+/node_modules
+/.idea
+/dist
+/build
+stats.json
```

Comment:
For what you comitted asset folder with cat.jpg and polo.png????

Reply:
done